orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/UpdateClientConfigRequest.java

```diff
@@ -67,6 +67,10 @@ public class UpdateClientConfigRequest {
     @Expose
     private String idTokenSigningAlgorithm;
 
+    @SerializedName("identity_verification_supported")
+    @Expose
+    private Boolean identityVerificationSupported;
+
```

Comment:
Do we need to make this default to false? If we ship this as-is what impact does it have on the live SSE?

Reply:
Discussed and agreed to add some unit tests